build.bash

```diff
@@ -34,6 +34,9 @@ cp -r "./src/main/resources" "$distResourcesPath" >/dev/null
 # Write executable into the distributable
 echo "Writing Script..."
 echo "java\bin\java.exe --module-path javafx\lib --add-modules javafx.base,javafx.controls,javafx.fxml,javafx.graphics -jar $jarName" >dist/SoundInvestments.bat
-echo "./java/bin/java.exe --module-path javafx\lib --add-modules javafx.base,javafx.controls,javafx.fxml,javafx.graphics -jar $jarName" >dist/SoundInvestments.bash
+echo "#! /usr/bin/env bash
+cd -- \$(dirname \$BASH_SOURCE)
+java --module-path javafx/lib --add-modules javafx.base,javafx.controls,javafx.fxml,javafx.graphics -jar $jarName" >dist/SoundInvestments.command
```

Comment:
The idea behind this script is, that it bundles the relevant parts of java and javafx with it. That's why the path was relative, pointing at the java-executable that was packed in the distributable. When you just write `java` at the beginning, it requires the user to hav java downloaded and set in their path.
Since Kruse isn't gonna run it on a mac, this isn't a problem really, but just thought I'd point it out

Reply:
I see now, that totally makes more sense with the relative path. However, I get errors now, when I exchange `java` with `./java/bin/java` which is the correct path to my java command. Since it's only for me and Lars anyways, I think it's fine to just leave it.

Reply:
Weird but yeah, I agree, let's just leave it like that :thumbsup: